### ISSUE TITLE
Clarification on required `webdriverio` version

### ISSUE THREAD

**Issue by zachlysobey** [webdriverio/expect-webdriverio#1635]
In a project using this package, I've started getting install _warnings_ saying "`unmet peer webdriverio [...]`".

it seems recent versions of this package have a peer dependency on an *alpha* v9 release of wdio.

```json
  "peerDependencies": {
    "@wdio/globals": "^9.0.0-alpha.350",
    "@wdio/logger": "^9.0.0-alpha.350",
    "webdriverio": "^9.0.0-alpha.350"
  },
```
  
In [the README](https://github.com/webdriverio/expect-webdriverio/blob/main/README.md?plain=1#L20) however, it says:

> NOTE: [WebdriverIO](https://github.com/webdriverio/webdriverio) `v8.0.0` or higher is required!  

Is the *alpha* peer dep a mistake? Or should the README be updated?

**Comment by christian-bromann** [comment]
> Is the _alpha_ peer dep a mistake? Or should the README be updated?

The readme should be updated , mind taking a stab at this?

**Comment by zachlysobey** [comment]
Addressed via #1643

```diff
-NOTE: [WebdriverIO](https://github.com/webdriverio/webdriverio) `v8.0.0` or higher is required!
+NOTE: [WebdriverIO](https://github.com/webdriverio/webdriverio) `v9.0.0-alpha.350` or higher is required!
```

Lemme know if that looks good
